src/sentry/integrations/vsts/client.py

```diff
@@ -213,11 +213,25 @@ def get_project(self, instance, project_id):
         )
 
     def get_projects(self, instance):
-        # TODO(dcramer): VSTS doesn't provide a way to search, so we're
-        # making the assumption that a user has 100 or less projects today.
-        return self.get(
-            VstsApiPath.projects.format(instance=instance), params={"stateFilter": "WellFormed"}
-        )
+        limit = 100  # 100 is max
```

Comment:
I thought I saw somewhere on the docs that 200 was the max value for `$top` but now I can't seem to find it, but ill try looking again.